src/Password/Bcrypt.php

```diff
@@ -68,41 +67,35 @@ public function __construct($options = [])
      * Bcrypt
      *
      * @param  string $password
+     * @param  boolean $cryptuse
      * @throws Exception\RuntimeException
      * @return string
      */
-    public function create($password)
+    public function create($password, $cryptuse = false)
```

Comment:
That was a test parameter that I used during the development. I forgot to remove it :)